h2/connection.py

```diff
@@ -1733,10 +1733,13 @@ def _receive_window_update_frame(self, frame):
         )
 
         if frame.stream_id:
-            stream = self._get_stream_by_id(frame.stream_id)
-            frames, stream_events = stream.receive_window_update(
-                frame.window_increment
-            )
+            try:
+                stream = self._get_stream_by_id(frame.stream_id)
+                frames, stream_events = stream.receive_window_update(
+                    frame.window_increment
+                )
+            except StreamClosedError:
+                return [], []
```

Comment:
Just to keep the state machine logic happy, we should `return [], events` here.

Reply:
Ok, done.